examples/all-combined/README.md

```diff
@@ -78,3 +78,7 @@ The example is split into separated templates related with their specific scope:
 The test is currently using the existing SecretManager region to deploy the VPC and the cluster if this value is not null. Instead if null it follows what set through `var.region`
 
 This logic is achieved through the local `sm_region` variable that is then used to create resources.
+
+### Note for the example maintainers
```

Comment:
This comment should go in the test and not the example. These resources are not used directly in the example, only the test.